src/vue-components/morph-inner-v1.vue

```diff
@@ -105,7 +105,7 @@
   </div><!--alpheios-morph__dictentry-->
 </template>
 <script>
-  import { LanguageModelFactory, GrmFeature } from 'alpheios-data-models'
+  import { LanguageModelFactory, Feature, GrmFeature } from 'alpheios-data-models'
```

Comment:
It seems that GrmFeature is not used any more (which is a great thing!) so we can remove it from import.

Reply:
ok, I had to make a small change to the tests to get rid of it, but that made sense. Thanks!